README.md

```diff
@@ -204,7 +205,9 @@ Optional Parameters:
 | target      | Number | A resource address to target                                                                                       |
 | noColor     | Any    | If this property exists, the -no-color flag is set                                                                 |
 | tfRootDir   | String | A terraform config directory to destroy; defaults to current directory                                             |
+| plan        | String | A terraform plan to destroy; if both plan and tfRootDir are specified, only plan is used                           |
```

Comment:
Shouldn't plan only be for the 'apply' goal?

Reply:
Removed plan and refreshState in tf:destroy which is not required for terraform destroy.